pkg/apis/3scale/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,6 +1,6 @@
 // +build !ignore_autogenerated
 
-// Code generated by operator-sdk. DO NOT EDIT.
+// Code generated by deepcopy-gen. DO NOT EDIT.
```

Comment:
why does this keep changing?

Reply:
no idea. I'm on 0.8.0 of the operator-sdk, maybe someone else is using an older version?